x-pack/platform/plugins/shared/fleet/public/components/transform_install_as_current_user_callout.tsx

```diff
@@ -28,12 +29,24 @@ export const getNumTransformAssets = (assets?: PackageInfo['assets']) => {
 export const TransformInstallWithCurrentUserPermissionCallout: React.FunctionComponent<{
   count: number;
 }> = ({ count }) => {
+  const { docLinks } = useStartServices();
   return (
     <EuiCallOut color="primary" iconType="iInCircle">
       <FormattedMessage
         id="xpack.fleet.createPackagePolicy.transformInstallWithCurrentUserPermissionCallout"
-        defaultMessage="This package has {count, plural, one {one transform asset} other {# transform assets}} which will be created and started with the same roles as the user installing the package."
-        values={{ count }}
+        defaultMessage="This package has {count, plural, one {one} other {#}} {docsLink} which will be created and started with the same roles as the user installing the package."
+        values={{ count, docsLink: (
+          <EuiLink
```

Comment:
Should that link be external and with `target=_blank` for consistency with other doc links?

Reply:
Oh! Yes! Updating now